subo/command/push.go

```diff
@@ -14,14 +14,15 @@ import (
 
 var validPublishTypes = map[string]bool{
 	"bindle": true,
+	"docker": true,
 }
 
 //PushCmd packages the current project into a Bindle and pushes it to a Bindle server.
 func PushCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "push",
-		Short: "publish the project",
-		Long:  "publish the current project to a remote server",
+		Short: "publish a project",
+		Long:  "publish the current project to a remote server (Docker, Bindle, etc.)",
```

Comment:
super nit: etc is potentially not needed here as docker and bindle are the only two places publishing can end up in

Reply:
Not for long :)